OmniDB/OmniDB_app/templates/OmniDB_app/base.html

```diff
@@ -408,9 +408,9 @@
                 		<img src="{{url_folder}}/static/OmniDB_app/images/firebird_medium.png" title="Firebird"/>
                 		<img src="{{url_folder}}/static/OmniDB_app/images/access_medium.png" title="Microsoft Access"/>-->
                 	</div>
-                	<div style="margin: 20px;"><a onclick="showWebsite('OmniDB-NG', 'http://omnidb-ng.org')" >omnidb-ng.org</a></div>
+                	<div style="margin: 20px;"><a onclick="showWebsite('PgManage', 'http://pg-manage.org')" >pg-manage.org</a></div>
```

Comment:
There is no such website. Please change the url to https://github.com/commandprompt/pgmanage for now.